Add -lgif when linking.

This patch from the Ubuntu package by Matthias Klose <doko@ubuntu.com>. 

--- a/fontforge/Makefile.dynamic.in
+++ b/fontforge/Makefile.dynamic.in
@@ -19,7 +19,7 @@
 X_11LIB = @X_11LIB@
 X_EXTRA_LIBS = @X_EXTRA_LIBS@
 X_PRE_LIBS = @X_PRE_LIBS@
-STATIC_LIBS = @STATIC_LIBS@
+STATIC_LIBS = @STATIC_LIBS@ -lgif
 
 FF_VERSION=1
 FF_REVISION=0
